.github/workflows/pull-request.yml

```diff
@@ -31,14 +41,25 @@ jobs:
       - uses: ./.github/actions/setup-zsh
       - uses: ./.github/actions/short-sha
 
-      - name: Build base env
+      - name: Determine Environment
+        id: set_env
         run: |
-          source uhd.sh
-          uhd terraform init:layer 20-app
-          uhd terraform apply:layer 20-app ci-$SHORT_SHA
-        shell: zsh {0}
+          if [[ "${{ github.head_ref }}" == *"auth-dev"* ]]; then
+            echo "TARGET_ENV=auth-dev" >> $GITHUB_ENV
+            echo "TARGET_ENV=auth-dev" >> $GITHUB_OUTPUT
```

Comment:
Why shouldn't be considering or using the auth-dev env within the pull-request workflow

Reply:
Agreed, removed the `if` statement code logic